UniversalScannerService/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+﻿{

```

Comment:
I would like to have one single running file at launch, as it is a very small and easy tool designed to scan network for device on customer site.